### PR TITLE
watcher beam projector [ready for merge]

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: 

:cl: zamolxius
add: added watcher beam projector
imageadd: added sprites
/:cl:

[why]: 

32fe82905201d9e9fbc91576c3d2d3ea

051fef2cce121101ca5b25f3b451f25d

1d2807e503ea3ed0220e2399db49c55f

Crafted with 2 diamonds, 2 watcher sinew and 3 bones, does 10 burn damage on hit and works like a ghetto temperature rifle on freeze (mitigated by warm clothing).
Doesn't dig very far, but still better than a pick. Crafting time is pretty long.
Has only 10 shots, plasma ore recharges 5 and plasma bar recharges 10. 

Good equalizer for ashwalkers, they can shoot you to slow then chase you down with spears.

### CODE REVIEW

**Comment by Jujumatic** [comment]
cool.

let me know when it's done.

also out of curiosity how will it be obtainable?

**Comment by ghost** [comment]
crafting,  diamonds,  bones and sinew
doesnt shoot too far and requires a lot of ammo

**Comment by ghost** [comment]
ready for review

**Comment by dreamcrab** [comment]
this looks neat and im willing to try it +8

**Review by yoyobatty — DISMISSED** [review]
Looks good to me +1

**Comment by ghost** [comment]
@Carbonhell

**Comment by Carbonhell** [comment]
Yeah sure

File: hippiestation/code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	desc = "A spiny, gruesome tool which reproduces the icy beam of a watcher, shattering rock and freezing individuals."
+	icon_state = "watcherprojector"
+	icon = 'hippiestation/icons/obj/guns/energy.dmi'
+	item_state = "watcherprojector"
+	lefthand_file = 'hippiestation/icons/mob/inhands/lefthand.dmi'
+	righthand_file = 'hippiestation/icons/mob/inhands/righthand.dmi'
+	ammo_type = list(/obj/item/ammo_casing/energy/plasma/watcher)
+	attack_verb = list("bashed", "stared down", "whacked", "smashed")
+	force = 10
+	can_charge = 0
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/projectiles/guns/energy/special.dm:12]: Is this intended to be used by ashwalkers and free golems? Then you may want to add 

`trigger_guard = TRIGGER_GUARD_ALLOW_ALL`

here somewhere

**Comment by ghost** [comment]
oh,  perfect, will do.  thank you

**Comment by ghost** [comment]
ready for merge

**Comment by yoyobatty** [comment]
Will you be adding what guyon suggested or no?

**Review by yoyobatty — APPROVED** [review]
Code looks good

**Comment by ghost** [comment]
oh but i added it

**Pull request merged**